schemas/v1/8_vm_messages.go

```diff
@@ -0,0 +1,46 @@
+package v1
+
+func init() {
+	patches.Register(
+		8,
+		`
+ CREATE TABLE {{ .SchemaName | default "public"}}.vm_messages (
+    height bigint NOT NULL,
+    state_root text NOT NULL,
+    cid text NOT NULL,
+	source text,
+    "from" text NOT NULL,
+    "to" text NOT NULL,
+    value numeric NOT NULL,
+    method bigint NOT NULL,
+    actor_code text NOT NULL,
+    exit_code bigint NOT NULL,
+    gas_used bigint NOT NULL,
+    params jsonb,
+	returns jsonb
+);
+ALTER TABLE ONLY {{ .SchemaName | default "public"}}.vm_messages ADD CONSTRAINT vm_messages_pkey PRIMARY KEY (height, state_root, cid, source);
+CREATE INDEX vm_messages_height_idx ON {{ .SchemaName | default "public"}}.vm_messages USING BRIN (height);
```

Comment:
I think BRIN indexing is dependent on write order, which is use-case specific. BTREE is the better generalized index here.
```suggestion
CREATE INDEX vm_messages_height_idx ON {{ .SchemaName | default "public"}}.vm_messages USING BTREE (height);
```

Reply:
Indeed, will use btree instead.